src/resources/BankAccounts.ts

```diff
@@ -71,49 +71,27 @@ export class BankAccounts extends CRUDResource {
 
     static routeBase = "/bank_accounts";
 
-    list(
-        data?: SendData<BankAccountsListParams>,
-        auth?: AuthParams,
-        callback?: ResponseCallback<ResponseBankAccounts>
-    ): Promise<ResponseBankAccounts> {
-        return this._listRoute()(data, callback, auth);
+    list(data?: SendData<BankAccountsListParams>, auth?: AuthParams): Promise<ResponseBankAccounts> {
+        return this._listRoute()(data, auth);
```

Comment:
maybe `this._listRoute()` doesn't need to be called each time? This could be a private field `this.#list = this._listRoute()`?
Maybe define a `getter` to make it lazy?

Reply:
Added a lazy load mechanic. Typescript does not really like the lazy getter thing 🤔 Or maybe I missed something?